### PR TITLE
Auto-fix style from historical style rules

### PR DESCRIPTION
**Pull request by RoryO**
Most changes from `eslint -fix`. Most significant are

1. Change bracket spacing from always require inner padding to never require. Consistent with most of existing code
2. Fixes mixes of tabs and spaces
3. Changes snake_case variables to camelCase, except in the one object designed for remote.
4. Fixes missing semicolons
5. Changes adding spacing around operators

I've poked around and everything appears to work properly.

Though it's mostly bikeshedding, having consistent code style helps new contributors get started on the codebase faster!

### CODE REVIEW

**Comment by patriciogonzalezvivo** [comment]
hi! thanks for this! it's great!
Any way to leave snake_case as they are? the `this.options.[property_names]` are use to set and customize the editor in different part of thebookofshaders.com editor.thebookofshaders.com editor.pixelspiritdeck.com

**Comment by RoryO** [comment]
Sorry. I will do that.

File: src/js/GlslEditor.js [hunk 1]
```diff
@@ -74,8 +74,8 @@ export default class GlslEditor {
             this.options.imgs = [];
         }
 
-        if (this.options.displayMenu === undefined) {
-            this.options.displayMenu = true;
```
Comment (RoryO) [inline src/js/GlslEditor.js]: I can't find any reference to this in here or the book of shaders

**Comment by RoryO** [comment]
Okay, reverted the options form camel case back to underscore.

**Pull request merged**